### DISCUSSION TITLE
Upcoming SQL schema changes

### DISCUSSION THREAD

**Discussion by jovinson-ms** [microsoft/dicom-server#1420] [General]
Post your upcoming SQL schema version changes here, and remove once the change is checked in.
- New version number
- Size (S/M/L)

**Comment by pengchen0692** [comment]
15 / S
* Dicom-Paas-CI Deadlock
* AddExtendedQueryTag (PROD) Deadlock

https://github.com/microsoft/dicom-server/pull/1402

**Comment by jovinson-ms** [comment]
Request cancel: 14 / M
